Kudu.Agent/Startup.cs

```diff
@@ -0,0 +1,42 @@
+// ------------------------------------------------------------------------------
+//  <copyright file="Startup.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+//  </copyright>
+// ------------------------------------------------------------------------------
+
+using System.IO;
+using Kudu.Agent.Security;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Kudu.Agent
+{
+    public class Startup
+    {
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+
+            // Use basic authentication to validate user requests
+            app.UseMiddleware<BasicAuthHelper>();
+
+
+            app.UseRouting()
+               .UseEndpoints(endpoints =>
+               {
+                   // Map attribute routed controllers.
+                   endpoints.MapControllers();
+                   // Map custom route for canary inside main container.
+                   endpoints.MapGet("/mscanary471987.php", async context =>
```

Comment:
Is it a temporary hardcoding?

Reply:
@jvano do you have the answer to this? I'm not entirely sure what is happening here.

Reply:
you can remove it, in the PoC that I shared with you, I was considering having a canary endpoint inside the container. I might do something similar but not now, so feel free to remove the mscanary endpoint